src/context/AppContext.tsx

```diff
@@ -45,6 +48,7 @@ export const AppContextProvider = ({ children }: { children: ReactNode }) => {
         const query = apolloClient.query({
           query: gql(getCurrentUser),
           variables: { address },
+          fetchPolicy: 'network-only',
```

Comment:
Can we change this to `cache-and-network` ?

I think it will have the same result that you're looking for, while also allow us to leverage some of Apollo's client side caching _(think instances where you log out / log in with the same wallet and user)_

Reply:
Not allowing me because of
`export declare type FetchPolicy = 'cache-first' | 'network-only' | 'cache-only' | 'no-cache' | 'standby';`
and 
`Invariant Violation: The cache-and-network fetchPolicy does not work with client.query, because client.query can only return a single result. Please use client.watchQuery to receive multiple results from the cache and the network, or consider using a different fetchPolicy, such as cache-first or network-only.`